### PR TITLE
Add murmurhash3js@3.0.1 w/ npm auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10705

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/pid/murmurHash3js
 * Official website (optional, not the repository): https://pid.github.io/murmurHash3js/
 * NPM package url (optional): https://www.npmjs.com/package/murmurhash3js
 * License and its reference: [MIT](https://github.com/pid/murmurHash3js/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 1
   - Count of stars: 6
   - Count of forks: 10
 * NPM download stats (optional):
   - Downloads in the last day: 343
   - Downloads in the last week: 2,339
   - Downloads in the last month: 11,264

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 Please help me review this pull request, thank y

File: ajax/libs/murmurhash3js/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "homepage": "https://pid.github.io/murmurHash3js/",
+  "description": "A javascript implementation of MurmurHash3's x86 hashing algorithms.(Browser&Server)",
+  "keywords": [
+    "murmurhash3",
+    "hash"
+  ],
+  "author": {
+    "name": "Karan Lyons",
+    "email": "karan@karanlyons.com",
+    "url": "http://karanlyons.com"
```
Comment (sufuf3) [inline ajax/libs/murmurhash3js/package.json]: https://www.karanlyons.com/

**Comment by extend1994** [comment]
@sufuf3 I modified the `url`, please help me review this pull request again, thank you.

File: ajax/libs/murmurhash3js/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/pid/murmurHash3js.git"
+  },
+  "npmName": "murmurhash3js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "murmurhash3js.min.js"
```
Comment (pvnr0082t) [inline ajax/libs/murmurhash3js/package.json]: Why not use `murmurHash3js*`?

File: ajax/libs/murmurhash3js/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      ]
+    },
+    {
+      "basePath": "lib",
+      "files": [
+        "murmurHash3js.js"
```
Comment (pvnr0082t) [inline ajax/libs/murmurhash3js/package.json]: Why not use `murmurHash3js*`?

Comment (hare1039) [inline ajax/libs/murmurhash3js/package.json]: Which is better for rules?
`{LIB_NAME}*.(map|js|css)`
`{LIB_NAME}*`
or some greater rule?
I'm a little bit confuse about this.

Comment (pvnr0082t) [inline ajax/libs/murmurhash3js/package.json]: In this case, we can just use `murmurHash3js*`. If there are some files we don't need mixed in the same directory, we need to use more restricted rule.

Comment (extend1994) [inline ajax/libs/murmurhash3js/package.json]: @pvnr0082t They are capital "H" and lower-case "h". That's why I did't use the rule you mentioned. If the PR is okay except this, then I update the branch.

Comment (pvnr0082t) [inline ajax/libs/murmurhash3js/package.json]: @extend1994 Okay, but my point is to make it more flexible. We need to prepare for the future updating.

Comment (extend1994) [inline ajax/libs/murmurhash3js/package.json]: @pvnr0082t Yes I know and thank you :+1: 
I updated the branch, please help me review it, thanks.

Comment (pvnr0082t) [inline ajax/libs/murmurhash3js/package.json]: @extend1994 It seems that you just rebased the branch onto the latest master branch. Anything updated?

Comment (extend1994) [inline ajax/libs/murmurhash3js/package.json]: @pvnr0082t No, there is no update. I was trying to say I rebased it, but I said I updated it. It was my fault, sorry!

Comment (pvnr0082t) [inline ajax/libs/murmurhash3js/package.json]: So, what's the reason we don't use much flexible rule `murmurHash3js*`?

Comment (extend1994) [inline ajax/libs/murmurhash3js/package.json]: @pvnr0082t So I should make it like this?
```
{
      "basePath": "",
      "files": [
        "murmurhash3js*"
      ]
    },
    {
      "basePath": "lib",
      "files": [
        "murmurHash3js*"
      ]
}
```

Comment (pvnr0082t) [inline ajax/libs/murmurhash3js/package.json]: @extend1994 Yes. I think it's better. :+1:

**Comment by extend1994** [comment]
@pvnr0082t I updated the rule, please help me review it, thank you.

**Pull request merged**